test/dialect/postgresql/test_types.py

```diff
@@ -3180,6 +3180,31 @@ def test_interval_coercion_literal(self):
         expr = column("bar", postgresql.INTERVAL) == datetime.timedelta(days=1)
         eq_(expr.right.type._type_affinity, types.Interval)
 
+    def test_interval_literal_processor(self, connection):
+        result = connection.execute(
+            select(
+                literal(
+                    datetime.timedelta(days=1, minutes=3, seconds=4),
+                    literal_execute=True,
```

Comment:
**mike bayer** (zzzeek) wrote:

this is the wrong form for parameters also

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4873